### PR TITLE
Add the latest static Ulster Scots translations

### PR DESCRIPTION
**Pull request by petechd**
## What is the context of this PR?
Adds the latest fixes from crowdin to the Ulster Scots static messages file.

## How to review
Check the translated strings match crowdin.

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by petechd** [comment]
[Load Ulster Scots PO files for NISRA](https://trello.com/c/hREqXUon/4416-load-ulster-scots-po-files-for-nisra)

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+msgid "If you need to give more feedback, please <a href='{contact_us_url}'>contact us</a>."
+msgstr "Gin ye need tae gie mair feedbak, plaise<a href='{contact_us_url}'>get tha houl o us</a>."
+
+#: app/routes/errors.py:161
+msgid "Sorry, there was a problem sending the access code"
+msgstr "Thair wus a problem sennin tha ingang code"
+
+#: app/routes/errors.py:167
+msgid "You can try to <a href='{retry_url}'>request a new access code again</a>."
+msgstr "Ye can try tae<a href='{retry_url}'>ax fur a new ingang code agane</a>."
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:300]:
```suggestion
msgstr "Ye can try tae <a href='{retry_url}'>ax fur a new ingang code agane</a>."
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+msgid "If this problem keeps happening, please <a href='{contact_us_url}'>contact us</a> for help."
+msgstr "Gin this gies ye mair bother, plaise <a href='{contact_us_url}'> get tha houl o us</a>fur a pull oot."
+
+#: app/routes/errors.py:189
+msgid "Sorry, there was a problem sending the confirmation email"
+msgstr "Thair wus a problem sennin tha homologatin email"
+
+#: app/routes/errors.py:190
+msgid "You can try to <a href='{retry_url}'>send the email again</a>."
+msgstr "Ye can try tae<a href='{retry_url}'>sen tha email agane</a>."
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:312]:
```suggestion
msgstr "Ye can try tae <a href='{retry_url}'>sen tha email agane</a>."
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+
+#: app/routes/errors.py:211 templates/errors/403.html:3
+#: templates/errors/403.html:6 templates/errors/submission-failed.html:5
+#: templates/errors/submission-failed.html:8
+msgid "Sorry, there is a problem"
+msgstr "We'r hairt-sair, thar's a pall"
+
+#: app/routes/errors.py:212
+msgid "You can try to <a href='{retry_url}'>submit your feedback again</a>."
+msgstr "Ye can try tae<a href='{retry_url}'>pit in yer feedbak agane</a>."
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:322]:
```suggestion
msgstr "Ye can try tae <a href='{retry_url}'>pit in yer feedbak agane</a>."
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
-#, python-format
-msgid "Your answers were submitted for <span>%(ru_name)s</span> (%(trading_as_name)s) on %(submitted_time)s"
-msgstr "Yer repones wus pit in fur <span>%(ru_name)s</span>(%(trading_as_name)s) oan %(submitted_time)s"
+#: app/views/handlers/individual_response.py:242
+msgid "Cannot answer questions for others in your household"
+msgstr "Cannae mak repone fur speirins anent ithers in yer hoosehaud"
+
+#: app/views/handlers/individual_response.py:308
+msgid "How would you like <em>{person_name}</em> to receive a separate census?"
+msgstr "Hoo wud ye lake<em>{person_name}</em>tae get anither heid-coont?"
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:517]:
```suggestion
msgstr "Hoo wud ye lake <em>{person_name}</em> tae get anither heid-coont?"
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+#: app/views/handlers/individual_response.py:580
+msgid "A letter with an individual access code will be sent to your registered household address"
+msgstr "A letter wi a sing'l ingang code wull be sent tae yer offeecial hoosehaud address"
 
-#: templates/errors/500.html:14
-msgid "Try again later."
-msgstr "Ettle at it agane eftèrhin."
+#: app/views/handlers/individual_response.py:587
+msgid "The letter will be addressed to <strong>Individual Resident</strong> instead of the name provided"
+msgstr "Tha letter wull be made oot tae <strong>Sing'l Resydent</strong>insteid o tha name gien tae us"
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:585]:
```suggestion
msgstr "Tha letter wull be made oot tae <strong>Sing'l Resydent</strong> insteid o tha name gien tae us"
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+#: templates/confirmation-email-sent.html:16
+msgid "A confirmation email has been sent to {email}"
+msgstr "An homologation email haes bin sent til {email}"
 
-#: templates/partials/question.html:39
-msgid "cleared"
-msgstr "redd"
+#: templates/confirmation-email-sent.html:25
+msgid "The email will be sent from <em>census.2021@notifications.service.gov.uk</em>"
+msgstr "Tha email wull be sent frae<em>census.2021@notifications.service.gov.uk</em>"
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:694]:
```suggestion
msgstr "Tha email wull be sent frae <em>census.2021@notifications.service.gov.uk</em>"
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+#: templates/errors/session-expired.html:7
+msgid "To help protect your information we have timed you out"
+msgstr "Tae help bield yer wittins we hae timed ye oot"
 
-#: templates/static/accessibility.html:83
-msgid "Non-descriptive form elements. On the ‘Check your answers’ summary page there is an option to “change” items, but the link does not describe where it goes to. Fails on WCAG 2.1 A number 1.3.1."
-msgstr ""
+#: templates/errors/session-expired.html:8
+msgid "You will need to <a href='{url}'>enter your 16-character access code</a> again to continue your census."
+msgstr "Ye maun<a href='{url}'>pit in yer 16-chairacter ingang code</a>agane tae conteenye wi yer heid-coont."
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:927]:
```suggestion
msgstr "Ye maun <a href='{url}'>pit in yer 16-chairacter ingang code</a> agane tae conteenye wi yer heid-coont."
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+#: templates/errors/submission-complete.html:7
+msgid "Your census has been submitted"
+msgstr "Yer heid-coont haes bin pit in"
 
-#: templates/static/accessibility.html:91
-msgid "On the ‘Is this address correct?’ page, the title may not accurately reflect the content on the page for audio users. The ‘Submit survey’ page does not have a page title. Fails on WCAG 2.1 A number 2.4.2."
-msgstr ""
+#: templates/errors/submission-failed.html:9
+msgid "You can try to <a href='{url}'>submit your census again</a>"
+msgstr "Ye can try tae<a href='{url}'>pit in yer feedbak agane</a>"
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:943]:
```suggestion
msgstr "Ye can try tae <a href='{url}'>pit in yer feedbak agane</a>"
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
 
-#: templates/static/accessibility.html:93
-msgid "Timeout issue. Users are not informed of a time limit and service times out without warning. During the validation process there is no return link on the session timeout page. Fails on WCAG 2.1 A number 2.2.1 and AAA number 2.2.6."
-msgstr ""
+#: templates/errors/submission-failed.html:10
+msgid "If this problem keeps happening, please <a href='{url}'>contact us</a> for help."
+msgstr "Gin this gies ye mair bother, plaise <a href='{url}'> get tha houl o us</a>fur a pull oot."
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:947]: Gin this gies ye mair bother, plaise <a href='{url}'> get tha houl o us</a> fur a pull oot.

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 10]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+#: templates/individual_response/interstitial.html:13
+msgid "Request separate census"
+msgstr "Ax fur anither heid-coont"
 
-#: templates/static/accessibility.html:113
-msgid "How we tested this questionnaire"
-msgstr ""
+#: templates/individual_response/question.html:9
+msgid "To request a census in a different format or for further help, please <a href='{contact_us_url}'>contact us</a>"
+msgstr "Tae ax fur a heid-coont in anither lay-oot, or fur mair help, plaise<a href='{contact_us_url}'>get tha houl o us</a>"
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:987]:
```suggestion
msgstr "Tae ax fur a heid-coont in anither lay-oot, or fur mair help, plaise <a href='{contact_us_url}'>get tha houl o us</a>"
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 11]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+#: templates/layouts/_base.html:61
+msgid "Tell us whether you accept cookies"
+msgstr "Tell us gin ye accept cookies"
 
-#: templates/static/accessibility.html:119
-msgid "user testing with the public on different devices, different locations, in their homes with various assistive technologies including free and paid versions"
-msgstr ""
+#: templates/layouts/_base.html:62
+msgid "We use <a href='{cookie_settings_url}'>cookies to collect information</a> about how you use census.gov.uk. We use this information to make the website work as well as possible and improve our services."
+msgstr "We uise<a href='{cookie_settings_url}'>cookies tae gether wittins </a>aboot hoo ye uise census.gov.uk We uise these wittins tae mak tha wabsteid wark as weel as able an better oor sarvice."
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:999]:
```suggestion
msgstr "We uise <a href='{cookie_settings_url}'>cookies tae gether wittins </a> aboot hoo ye uise census.gov.uk We uise these wittins tae mak tha wabsteid wark as weel as able an better oor sarvice."
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 12]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
 
-#: templates/static/accessibility.html:120
-msgid "testing with charities, such as Scope and RNIB in their labs"
-msgstr ""
+#: templates/layouts/_base.html:63
+msgid "You’ve accepted all cookies. You can <a href='{cookie_settings_url}'>change your cookie preferences</a> at any time."
+msgstr "Ye hae acceptit aa tha cookies. Ye can <a href='{cookie_settings_url}'> cheynge yer cookie preferences</a>onietim ye lake."
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:1003]:
```suggestion
msgstr "Ye hae acceptit aa tha cookies. Ye can <a href='{cookie_settings_url}'> cheynge yer cookie preferences</a> onietim ye lake."
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 13]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+#: templates/partials/feedback-call-to-action.html:9
+msgid "Give feedback"
+msgstr "Gie feedbak"
 
-#: templates/static/privacy.html:10
-msgid "The information you provide when you fill in your census rehearsal questionnaire is protected by law."
-msgstr "Tha wittins ye gie whanivver ye fïll in yer heid-coont preein spierins is fendit wi tha laa."
+#: templates/partials/individual-response-guidance.html:24
+msgid "You can <em>share your household access code</em> with the people you live with so they can complete their own sections."
+msgstr "Ye can <em> gie oot yer hoosehaud ingang code</em>wi tha fowk wha leeve wi ye sae they can fill oot thair ain lay- oots."
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:1071]:
```suggestion
msgstr "Ye can <em> gie oot yer hoosehaud ingang code</em> wi tha fowk wha leeve wi ye sae they can fill oot thair ain lay- oots."
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 14]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+#: app/views/handlers/individual_response.py:403
+msgid "Send individual access code"
+msgstr "Sen sing'l ingang code"
 
-#: templates/errors/403.html:6
-msgid "Sorry, there is a problem"
-msgstr "We'r hairt-sair, thar's a pall"
+#: app/views/handlers/individual_response.py:434
+msgid "How would you like to answer <em>{person_name_possessive}</em> questions?"
+msgstr "Hoo wud ye lake tae mak repone fur<em>{person_name_possessive}</em> speirins?"
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:553]:
```suggestion
msgstr "Hoo wud ye lake tae mak repone fur <em>{person_name_possessive}</em> speirins?"
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 15]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+#: templates/individual_response/confirmation-text-message.html:17
+msgid "The text message with an individual access code should arrive soon for them to complete their own census"
+msgstr "Tha text message wi a sing'l ingang code shud land shane eneuch fur thaim tae fill oot thair ain heid-coont"
 
-#: templates/static/accessibility.html:103
-msgid "Some headings do not follow a logical hierarchical structure. Fails on WCAG AAA 2.1 number 2.4.10."
-msgstr ""
+#: templates/individual_response/confirmation-text-message.html:26
+msgid "The text will be sent from <em>Census2021</em>"
+msgstr "Tha text wull be sent frae<em>Heid-coont 20an21</em>"
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:967]:
```suggestion
msgstr "Tha text wull be sent frae <em>Heid-coont 20an21</em>"
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 16]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
+#: templates/census-thank-you.html:32
+msgid "Your individual census has been submitted for <strong>{display_address}</strong>"
+msgstr "Yer sing'l heid-coont haes bin pit in fur <strong>{display_address}</strong>"
 
-#: templates/layouts/configs/_footer.html:48
-msgid "Cookies"
-msgstr "Cookies"
+#: templates/census-thank-you.html:34
+msgid "Your census has been submitted for the household at <strong>{display_address}</strong>"
+msgstr "Yer heid-coont haes bin pit in fur tha hoosehaud at<strong>{display_address}</strong>"
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:662]:
```suggestion
msgstr "Yer heid-coont haes bin pit in fur tha hoosehaud at <strong>{display_address}</strong>"
```

File: app/translations/eo/LC_MESSAGES/messages.po [hunk 17]
```diff
@@ -2,1124 +2,1300 @@ msgid ""
...
 
-#: templates/layouts/configs/_footer.html:52 templates/static/privacy.html:9
-msgid "Privacy and data protection"
-msgstr "Hiddlins an gien taxt hainin"
+#: templates/census-thank-you.html:36
+msgid "Your census has been submitted for the accommodation at <strong>{display_address}</strong>"
+msgstr "Yer heid-coont haes bin pit in fur tha dwellin-place at<strong>{display_address}</strong>"
```
Comment (berroar) [inline app/translations/eo/LC_MESSAGES/messages.po:666]:
```suggestion
msgstr "Yer heid-coont haes bin pit in fur tha dwellin-place at <strong>{display_address}</strong>"
```

File: app/translations/eo/LC_MESSAGES/messages.po (hunk 1 above)
Comment (petechd) [reply app/translations/eo/LC_MESSAGES/messages.po]: 👍

File: app/translations/eo/LC_MESSAGES/messages.po (hunk 2 above)
Comment (petechd) [reply app/translations/eo/LC_MESSAGES/messages.po]: 👍

**Comment by petechd** [comment]
All changes now added, thanks @berroar

**Review by MebinAbraham — APPROVED** [review]

**Review by berroar — APPROVED** [review]

**Pull request merged**